src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -4,7 +4,6 @@
 #include <string>
```

Comment:
The bulk of changes in this file are formatting only, which makes it hard to find the real changes. Can you revert the formatting changes?